panel/chat/interface.py

```diff
@@ -80,7 +80,7 @@ class ChatInterface(ChatFeed):
     )
 
     reset_on_send = param.Boolean(
-        default=False,
+        default=True,
```

Comment:
I believe the reference guide should be updated too then.

Reply:
I think the reference guide doesn't mention the default.